README.md

```diff
@@ -3,11 +3,13 @@
 
 ### Welcome!
 
-You might be asking yourself "What is Open Cluster Management", well... the org `github.com/open-cluster-management` is the upstream staging area for a new product to be introduced, named "Red Hat Advanced Cluster Management for Kubernetes (`RHACM4K` pronounced \`rack-um-4k\` or for short `RHACM` pronounced \`rack-um\`)".
+You might be asking yourself "What is Open Cluster Management", well... the org `github.com/open-cluster-management` is the upstream staging area for a new product to be introduced, named "Red Hat Advanced Cluster Management for Kubernetes (`RHACM4K` pronounced \`rack-um-4k\` or for short `RHACM` pronounced \`rack-um\`)". View the Red Hat Advanced Cluster Management for Kubernetes architecture diagram:
```

Comment:
@tpouyer I have a couple of suggestions and questions for the first paragraph. I want to suggest that you update the first paragraph to the following:

You might be asking yourself,  "What is Open Cluster Management", well it is the `github.com/open-cluster-management`  org that is the upstream staging area for a new product to be introduced: "Red Hat Advanced Cluster Management for Kubernetes" (`RHACM4K` pronounced \`rack-um-4k\` or for short `RHACM` pronounced \`rack-um\`). View the Red Hat Advanced Cluster Management for Kubernetes architecture diagram:

If you are not attempting that I would still like to suggest that you move the pronunciation outside of the quotation marks.


did you want the product name to standout? I suggest changing `"Red Hat Advanced Cluster Management for Kubernetes (`RHACM4K` pronounced \`rack-um-4k\` or for short `RHACM` pronounced \`rack-um\`)"` TO `**Red Hat Advanced Cluster Management for Kubernetes** (`RHACM4K` pronounced \`rack-um-4k\` or for short `RHACM` pronounced \`rack-um\`)`, so that the name is bold